Dockerfile

```diff
@@ -44,9 +44,12 @@ ENTRYPOINT [ "poetry", "run", "flask", "run", "--host", "0.0.0.0" ]
 
 
 FROM builder-base as production
-RUN poetry install --no-dev   
+#RUN poetry install --no-dev   
+RUN poetry config virtualenvs.create false --local && poetry install --no-dev
 COPY  ./todo_app  /app/todo_app
-ENTRYPOINT [ "poetry", "run", "gunicorn", "--bind", "0.0.0.0:5000", "todo_app.wsgi:app" ]
+ENV PORT = 5000
+ENTRYPOINT ["bash", "./todo_app/run.sh"]
+#ENTRYPOINT [ "poetry", "run", "gunicorn", "--bind", "0.0.0.0:5000", "todo_app.wsgi:app" ]
```

Comment:
Similarly to above, leaving this sort of code in quickly builds up cruft that makes it harder to reason about what's remaining - let's clear this out!

Reply:
ok

Reply:
Removed commented code